R/ui.R

```diff
@@ -36,6 +39,7 @@ style_pkg <- function(pkg = ".",
                       ...,
                       style = tidyverse_style,
                       transformers = style(...),
+                      filetype = c("r"),
```

Comment:
I don't see how this argument does affect the function call, i.e. it is not really used further downstream if I see that correctly.

Reply:
So was your intend to wait with merging anyways until #293 is resolved?

Reply:
Yeah this was just because the other functions (style_dir & prettify_any) were already inheriting params from here and I figured we'd be adding this to style_pkg later anyway.

Reply:
Ok, lets wait with merging then.